src/display/canvas.js

```diff
@@ -1454,7 +1449,8 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
     get isFontSubpixelAAEnabled() {
       // Checks if anti-aliasing is enabled when scaled text is painted.
       // On Windows GDI scaled fonts looks bad.
-      var ctx = document.createElement('canvas').getContext('2d');
+      var canvasFactory = new CanvasFactory();
```

Comment:
CanvasFactory must be created once per render() call and stored at CanvasGraphics.